lib/ansiblelint/generate_docs.py

```diff
@@ -45,18 +45,19 @@ def main():
         id_text = id_link.format(d['id'], d['classname'])
         grid.append([id_text, d['version_added'], d['shortdesc'], d['description']])
 
-    filename = '../../docs/docsite/rst/rules/default_rules.rst'
+    filename = Path(__file__).resolve().parents[2].joinpath(
+        "docs/docsite/rst/rules/default_rules.rst")
+
     with open(filename, 'w') as file:
         file.write(make_table(grid))
         _logger.info('%s file written', filename)
 
 
 def import_all_rules():
-    for module in list(os.walk('rules'))[0][2]:
-        if module == '__init__.py' or module[-3:] != '.py':
-            continue
-        module = 'rules.{}'.format(module[:-3])
-        importlib.import_module(module)
+    prefix = 'ansiblelint.rules.'
+    for importer, modname, ispkg in pkgutil.iter_modules(rules.__path__, prefix):
+        _logger.info("Found submodule %s (is a package: %s)", modname, ispkg)
+        __import__(modname)
```

Comment:
I'm pretty sure that `__import__` is not recommended and using `importlib` is better.